backend/tournesol/models/entity.py

```diff
@@ -72,6 +72,13 @@ class Meta:
         "from certified contributors",
     )
 
+    tournesol_score = models.FloatField(
+        null=False,
+        default=0,
+        help_text="Tournesol Score"
+        "Calculated from the criteria_score from certified contributors",
```

Comment:
Note for self:
- should be nullable
- null by default
- "le score de l'entité agrégée sur l'ensemble des critères et l'ensemble des utilisateurs pour un scrutin donnée"